src/ui/studio/recording/index.js

```diff
@@ -0,0 +1,64 @@
+//; -*- mode: rjsx;-*-
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+
+import { faCaretLeft } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Box, Button, Flex } from '@theme-ui/components';
+import { useCallback, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { useRecordingState } from '../../../recording-context';
+
+import { PromptAndProceed } from '../elements';
+
+import MediaDevices from './media-devices';
+import RecordingControls from './recording-controls';
+import Warnings from './warnings';
+
+export default function Recording(props) {
+  const { t } = useTranslation();
+  const state = useRecordingState();
+
+  useEffect(() => {
+    if (!(state.displayStream || state.userStream)) {
+      props.firstStep();
+    }
+  }, [props, state.displayStream, state.userStream]);
+
+  const handleRecorded = () => {
+    props.nextStep();
+  };
+
+  const backToSetupVideo = useCallback(() => {
+    props.firstStep();
+  }, [props]);
+
+  const BackButton = ({ handlePrev }) => (
+    <Button onClick={handlePrev}>
+      <FontAwesomeIcon icon={faCaretLeft} />
+      {t('back-button-label')}
+    </Button>
+  );
+
+  const overlayStyle = {
+    position: 'absolute',
+    left: 0,
+    right: 0,
```

Comment:
Children of a flex container shouldn't be `position: absolute` as they don't participate in the flex layout. I think these lines can be removed. Or why were they added? They make the top bar overlap the video. If that's indeed intended, I think it should be solved in a different manner.